### PR TITLE
Add new command BITFIELD_RO

### PR DESCRIPTION
**Pull request by git-hulk**
Refer: https://github.com/apache/kvrocks/pull/1990

### CODE REVIEW

**Comment by PragmaTwice** [comment]
Could we also add MOVE and EXPIRETIME?

File: docs/supported-commands.md [hunk 1]
```diff
@@ -157,14 +157,15 @@
...
-| BITFIELD | √                | v2.7.0        |      |
-| BITOP    | √                | v2.1.0        |      |
+| Command     | Supported OR Not | Since Version | Desc |
+| ----------- | ---------------- | ------------- | ---- |
+| GETBIT      | √                | v1.0.0        |      |
+| SETBIT      | √                | v1.0.0        |      |
+| BITCOUNT    | √                | v1.0.0        |      |
+| BITPOS      | √                | v1.0.0        |      |
+| BITFIELD    | √                | v2.7.0        |      |
+| BITFIELD_RO | √                | -             |      |
```
Comment (mapleFU) [inline docs/supported-commands.md:167]: Hmmm should we change to v2.8.0 or just leave `-`? I've no idea

Comment (PragmaTwice) [reply docs/supported-commands.md:167]: I think we can use `unstable` or `v2.8.0`

Comment (git-hulk) [reply docs/supported-commands.md:167]: Guess we can change them at once after releasing 2.8.0

**Comment by git-hulk** [comment]
> Could we also add MOVE and EXPIRETIME?

Sure, I have added the EXPIRETIME/PEXPIRETIME/ANALYZE command and MOVE has been added since 2.6.0

File: docs/supported-commands.md [hunk 2]
```diff
@@ -230,40 +233,40 @@ These commands are subcommands for `CLUSTER`, using as `CLUSTER INFO` etc.
...
+| TIME        | √                | v2.4.0        |                                                                                                                         |
+| STATS       | √                | v1.0.0        | dump the rocksdb statistics in JSON format                                                                              |
+| RESTORE     | √                | v2.6.0        | create the new key with Redis serialized value, now supports String/List/Hash/Set/ZSet data types                       |
+| COMPACT     | √                | v1.0.0        |                                                                                                                         |
+| RDB         | √                | v2.7.0        | RDB LOAD command is used to load RDB file from local path into Kvrocks                                                  |
+| ANALYZE     | √                | -             | use to export the perf/iostat of the current coomand                                                                    |
```
Comment (mapleFU) [inline docs/supported-commands.md:269]:
```suggestion
| ANALYZE     | √                | -             | use to export the perf/iostat of the current comnand                                                                    |
```

**Review by mapleFU — COMMENTED** [review]
Rest LGTM

File: docs/supported-commands.md (hunk 1 above)
Comment (PragmaTwice) [reply docs/supported-commands.md:169]: I prefer unstable instead of -

Comment (git-hulk) [reply docs/supported-commands.md]: unstable is better

**Review by mapleFU — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Pull request merged**